config/config-server/src/main/resources/cruise-config.xsd

```diff
@@ -220,11 +220,17 @@
   </xsd:complexType>
   <xsd:complexType name="configRepoGitType">
     <xsd:attribute name="url" type="xsd:string" use="required"/>
+    <xsd:attribute name="username" type="xsd:string"/>
```

Comment:
Is it possible to define common attributes as an abstract type?

Reply:
It should be, but not sure if it was worth doing in xsd :-/